tensorflow/core/platform/cus.h

```diff
@@ -60,22 +60,26 @@ struct cus {
   static float castValueToF32(const uint32_t& u);
 };
 
+typedef float (*cusToF32Type)(tensorflow::cus);
+typedef cus (*f32ToCusType)(float);
+
+extern "C" {
 
 inline float CastCusToF32(cus c) { return (float)(c); }
 inline cus CastF32ToCus(const float f) { return cus(f); }
```

Comment:
You can use __attribute__((always_inline)) instead of inline.
This will instruct clang to mark it as inlined function. But would still have external linkage.